src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/core/Util.java

```diff
@@ -27,6 +27,7 @@
   private static final Logger LOG = LogManager.getLogger(Util.class);
   public static final String METRICS_QUERY_URL = "/_opendistro/_performanceanalyzer/metrics";
   public static final String RCA_QUERY_URL = "/_opendistro/_performanceanalyzer/rca";
+  public static final String ACTIONS_QUERY_URL = "/_opendistro/_performanceanalyzer/actions";
```

Comment:
Can you create an issue to add this new URL to the readme with some documentation ?

Reply:
Added the Documentation.